src/DssAutoLine.sol

```diff
@@ -51,13 +52,27 @@ contract DssAutoLine {
     }
 
     /*** Administration ***/
-    function file(bytes32 ilk, bytes32 what, uint256 data) external auth {
-        if      (what == "on")    ilks[ilk].on   = uint8(data);
-        else if (what == "ttl")   ilks[ilk].ttl  = uint48(data);
-        else if (what == "line")  ilks[ilk].line = uint256(data);
-        else if (what == "gap")   ilks[ilk].gap  = uint256(data);
-        else revert("DssAutoLine/file-unrecognized-param");
-        emit File(ilk, what, data);
+
+    /**
+        @dev Add or update an ilk
+        @param ilk    Collateral type
+        @param line   Collateral maximum debt ceiling that can be configured [RAD]
+        @param gap    Amount of collateral to step [RAD]
+        @param ttl    Minimum time between increase [seconds]
+    */
+    function enableIlk(bytes32 ilk, uint256 line, uint256 gap, uint256 ttl) external auth {
+        require(ttl < uint48(-1), "DssAutoLine/invalid-ttl");
+        ilks[ilk] = Ilk(line, gap, 1, uint48(ttl), 0, 0);
+        emit Enable(ilk);
```

Comment:
Maybe we want to save `line`, `gap` and `ttl` as well?

Reply:
updated event

Reply:
Looks good!